examples/processors/extract-field-transform.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+# This script will start Conduit and set up a pipeline which demonstrates
+# a transform replacing a record's structured payload with one of its fields.
+
+# The pipeline consists of:
+# * a generator source, generating structured records
+# * a file destination (it writes to file_destination.txt)
+# * a processor with the built-in extract-field transform.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+DEST_FILE="$SCRIPT_DIR/file_destination.txt"
+> "$DEST_FILE"
+
+echo "Running Conduit..."
+docker run --rm  --name conduit -v "$DEST_FILE":/file_destination.txt -p 8080:8080 -d ghcr.io/conduitio/conduit:latest
```

Comment:
Since this pulls the latest Docker image, this won't work until a nightly build with this change is out. I've verified this with a local image.